hack/install.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# Local operator-sdk version
+LOCAL_OPERATOR_SDK_VERSION="$(operator-sdk version | head -n1 | cut -d" " -f3 | tr -d '",')"
```

Comment:
if you set the flags, then you should first check if operator-sdk exists (using which for example) then make sure that you don't have unset variables, so for example default the LOCAL_OPERATOR_SDK_VERSION to an empty string